packages/forms-web-app/src/controllers/projects/documents/controller.js

```diff
@@ -21,13 +24,20 @@ const getApplicationDocuments = async (req, res) => {
 		const pageFeatureToggles = featureToggles();
 		const pageDataObj = await pageData(case_ref);
 
-		const { documents, filters, pagination } = await searchDocuments(case_ref, query);
+		const { documents, examinationLibraryDocument, filters, pagination } = await searchDocuments(
+			case_ref,
+			query
+		);
 		const documentsView = getDocuments(documents);
+		const examinationLibraryDocumentHtml = getExaminationLibraryDocumentHtml(
```

Comment:
this should be returned by get documents and exported as two objects. 

This relates to documents so should come from there ?